php/Terminus/Instrument.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Terminus;
+use \ReflectionClass;
+use \Terminus\Request;
+use \Terminus\Collections\Bindings;
+
+class Instrument {
+  private $_id;
+  private $_attributes;
+  private $_bindings;
+
+  private $_user;
+
+  public function __construct(User $user, $data = null) {
```

Comment:
The model signature I'd like to use is `($attributes = new stdClass(), $options = array())`.  Though eventually I'd like to get the attributes to be an associative array rather than a stdClass instance (the reason for it being stdClass is because that's what the API wrapper returns and redoing all of that is a beast of a project)